### ISSUE TITLE
OpenTimelineIO text is too light

### ISSUE THREAD

**Issue by jminor** [AcademySoftwareFoundation/artwork#42]
The OTIO branding image here: https://github.com/AcademySoftwareFoundation/artwork/tree/main/projects/opentimelineio/horizontal/color
has text which is too light to read easily when used on a light background. For example on the ASWF main project page here: https://www.aswf.io/projects/

There is a darker version available here which should be used when the background is light: 
https://github.com/AcademySoftwareFoundation/OpenTimelineIO/blob/main/docs/_static/OpenTimelineIO%403xDark.png

**Comment by Abhii67534** [comment]
Please assign this issue to me  :)

**Comment by Abhii67534** [comment]
![OTIO LOGO](https://github.com/AcademySoftwareFoundation/artwork/assets/154100444/d97919a7-d303-411a-abd3-b0a839fb090d)

**Comment by Abhii67534** [comment]
Please confirm

**Comment by jminor** [comment]
Yes, that looks better. Can you make a pull request to update the files here? https://github.com/AcademySoftwareFoundation/artwork/tree/main/projects/opentimelineio/horizontal/color
